### PR TITLE
docs: FAQ for archive scan and remove local ignore file section

### PR DESCRIPTION
**Pull request by srinivasakartheek**
### Summary
- Added FAQ for new archive scan feature
- Remove outdated `.blubracket-ignore` related documentation
- Update API key generation image

### CODE REVIEW

File: content/faq/what-file-types-are-scanned/index.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+title: "What file types are scanned?"
```
Comment (misterbisson) [inline content/faq/what-file-types-are-scanned/index.md:2]: The title suggests it's comprehensive of all file types we scan, but the content appears limited to just the archive file types.

Perhaps change the title to "What archive file types are scanned?"

File: content/how-to/ignore-rules/local/index.md [hunk 2]
```diff
@@ -1,26 +0,0 @@
----
-title: "Configuring local ignore rules"
```
Comment (misterbisson) [inline content/how-to/ignore-rules/local/index.md:2]: Is there a replacement for this?

Comment (srinivasakartheek) [reply content/how-to/ignore-rules/local/index.md:2]: ignore.yaml is the replacement, CLI will read the rules if that file is present

File: content/faq/what-file-types-are-scanned/index.md (hunk 1 above)
Comment (srinivasakartheek) [reply content/faq/what-file-types-are-scanned/index.md:2]: We are saying `BluBracket scans for code risks in all text encoded files`. Do you want it to be highlighted?

Comment (misterbisson) [reply content/faq/what-file-types-are-scanned/index.md:2]: Yeah, it definitely got lost.

**Review by misterbisson — DISMISSED** [review]

**Review by misterbisson — APPROVED** [review]

**Pull request merged**